### ISSUE TITLE
Extensions Tab

### ISSUE THREAD

**Issue by Tom25a** [Lord-Kamina/SwiftDefaultApps#27]
Hi, sorry for this silly question but nobody asked (or I didn't find) : 
Where can I find the extensions list ?

**Comment by Lord-Kamina** [comment]
There is none because macOS doesn't really use them anymore. Everything is based on UTIs behind the hood.

**Comment by Tom25a** [comment]
Oh ?  When I open the info for a file, I have the choice to open file with this application, and Apply to all file with this extension. I'm under Mojave, maybe it change with Catalina.

**Comment by Lord-Kamina** [comment]
Yes, but all those APIs are deprecated. Extensions are associated to a UTI. What I need to do is implement a search field in the UTI tab, as somebody else requested

**Comment by Tom25a** [comment]
I read the report where someone ask for a search field, or a filter. It would be great.
Thank you very much for your answer and your great job.

**Comment by wdhwg001** [comment]
And it would also be great if we can show an extension field after the UTI so that we could know which extension this UTI controls. It would be even better if the search field is able to search by extensions.

**Comment by Lord-Kamina** [comment]
> And it would also be great if we can show an extension field after the UTI so that we could know which extension this UTI controls. It would be even better if the search field is able to search by extensions.

Correct me if I'm wrong, since I haven't actually devoted time to this in a while but I think extensions ARE shown in the UTI tab?

**Comment by wdhwg001** [comment]
Yes indeed, but it is not in the application tab, so it would be a bit confusing to browse by Applications: I need to go back to UTI tabs to confirm the meanings of UTIs.

I'm sorry for not making things clear.

**Comment by Lord-Kamina** [comment]
> Yes indeed, but it is not in the application tab, so it would be a bit confusing to browse by Applications: I need to go back to UTI tabs to confirm the meanings of UTIs.
> 
> I'm sorry for not making things clear.

I don't think there's space to put that info in the application tab; at least not without a considerable redesign of the UI

**Comment by wdhwg001** [comment]
Maybe we could put them in the hover hint? Or I would suggest that we may have a toggle to switch between showing UTI names or extensions.
